templates/Datashades-OpsWorks-CKAN-Stack.cfn.yml.j2

```diff
@@ -350,6 +341,161 @@ Resources:
       Roles:
         - !Ref WebInstanceRole
 
+  # Populate SSM Parameter Store with all the information needed to deploy.
+  # This should allow a blank instance to set itself up based only on the tags applied to it.
+
+  PublicTLDParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: !Sub "/config/CKAN/${Environment}/app/${ApplicationId}/public_tld"
```

Comment:
many of these could be pulled in chef from ssm param store direct then it would stay current on rebuilds when param store changes but cfn is not re-run to pull in the new values.

Reply:
That's what this is intended to do. The CloudFormation resource is populating an SSM Parameter Store value, which Chef can then retrieve.